Sources/SQL/Core/ConnectionProtocol.swift

```diff
@@ -36,13 +36,13 @@ public protocol ConnectionInfoProtocol {
     var username: String? { get }
     var password: String? { get }
     
-    init?(uri: URI)
+    init?(uri: URL)
 }
 
 public protocol ConnectionProtocol: class {
     associatedtype InternalStatus
     associatedtype Result: ResultProtocol
-    associatedtype Error: ErrorProtocol, CustomStringConvertible
+    associatedtype ErrorAndStringConvertible: Error, CustomStringConvertible
```

Comment:
I assume it's because `Error` is a reserved name by swift now. I'll change it to `ConnectionError`